test_file.php

```diff
@@ -1,6 +1,9 @@
 <?
 $foo = 1;
 
+$have_time = true;
+$have_money = false;
+$take_vacation = $have_time and $have_money;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace "and" with "&&". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://az.dkel7.jellyfish.local:8888/coding_rules#rule_key=php%3AS2010)